elasticdl/python/worker/task_data_service.py

```diff
@@ -83,6 +83,8 @@ def report_record_done(self, count, err_msg=""):
         if err_msg:
             self._failed_record_count += count
 
+        if not self._pending_tasks:
```

Comment:
When will this happen that worker reports record done but there are no pending tasks?

Reply:
Add a workaround comment here. I will debug it later.